tests/tools/kopia_runner/kopiarun.go

```diff
@@ -0,0 +1,97 @@
+// Package kopiarun wraps the execution of the kopia binary.
+package kopiarun
```

Comment:
Make sure the package path match the package name to avoid user (and IDE/tools) confusion.

The path should probably be `tests/tools/kopiarun` or maybe ``tests/tools/kopiarunner` and change the package name to `kopiarunner` to match.